Source/WebCore/css/parser/CSSParserImpl.cpp

```diff
@@ -651,6 +651,11 @@ RefPtr<StyleRuleKeyframes> CSSParserImpl::consumeKeyframesRule(CSSParserTokenRan
     AtomString name;
     if (nameToken.type() == IdentToken) {
         name = nameToken.value().toAtomString();
+        // From the CSS Values specification, keyframe names are not allowed to be
+        // CSS wide keywords. Nor can they be "default". And CSS Animations additionally
+        // excludes the "none" keyword.
```

Comment:
maybe clarify that this is only when the name is an identifier (it doesn't apply when the name is a string)

Reply:
It's in the branch guarded by `if (nameToken.type() == IdentToken) {`

Reply:
But I've added more to the comment.